controllers/questionController.js

```diff
@@ -0,0 +1,61 @@
+import {
+   save,
+   getAllQuestions,
+   getQuestionByID,
+   getQuestionsFromIdArray,
+   updateQuestionByID,
+   deleteQuestionByID
+} from '../models/questionModel';
+
+import AppError from '../errors/AppError';
+
+
+
+exports.addNewQuestion = async(req, res, next) => {
+   try {
+      const model = {
+         question: req.body.question,
+         correct: req.body.correct,
+         answers: req.body.answers,
+         type: req.body.type
+      }
+      const question = await save(model);
+
+      res.status(200).send({payload: question})
+   } catch(error) { next(new AppError(error.message ))};
```

Comment:
Please style it properly content of catch block should be on new line

Reply:
done